src/arabic.c

```diff
@@ -262,36 +259,35 @@ can_join(int c1, int c2)
  * Check whether we are dealing with a character that could be regarded as an
  * Arabic combining character, need to check the character before this.
  */
-    int
+int
 arabic_maycombine(int two)
 {
     if (p_arshape && !p_tbidi)
-	return (two == a_ALEF_MADDA
-		    || two == a_ALEF_HAMZA_ABOVE
-		    || two == a_ALEF_HAMZA_BELOW
-		    || two == a_ALEF);
+        return (two == a_ALEF_MADDA
+                || two == a_ALEF_HAMZA_ABOVE
+                || two == a_ALEF_HAMZA_BELOW
+                || two == a_ALEF);
     return FALSE;
 }
 
 /*
  * Check whether we are dealing with Arabic combining characters.
  * Note: these are NOT really composing characters!
  */
-    int
-arabic_combine(
-    int		one,	    // first character
-    int		two)	    // character just after "one"
+int
+arabic_combine(int one,        // first character
```

Comment:
The convention is to place the arguments in separate lines (one per line) if placing all of them in a single line will exceed 80 columns.  Another convention is to align the argument names by using a softtab.

Reply:
Is 80 max width in general?